client/boot/app.js

```diff
@@ -0,0 +1,60 @@
+// Initialize localStorage polyfill before any dependencies are loaded
+require( 'lib/local-storage' )();
+if ( process.env.NODE_ENV === 'development' ) {
+	require( 'lib/wrap-es6-functions' )();
```

Comment:
I would expect this to result in the wrappers still being part of the production bundle, just never executed. True?